<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>I2pSettings</class>
 <widget class="QWidget" name="I2pSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки i2p</string>
  </property>
  <property name="windowIcon">
   <iconset resource="source.qrc">
    <normaloff>:/mainapp/sources/i2p.png</normaloff>:/mainapp/sources/i2p.png</iconset>
  </property>
  <widget class="QPushButton" name="save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>99</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="binaryPath">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>309</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Путь до исполняемого файла i2p</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tunnelsConfPath">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>309</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Путь до конфига с тунелями i2p (tunnels.conf)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="httpProxy">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>311</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>4444</number>
   </property>
  </widget>
  <widget class="QLabel" name="i2pHttpPortLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>i2p.settings.http_proxy_port</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
